cgsmiles/tests/test_molecule_resolve.py

```diff
@@ -53,6 +58,16 @@ def test_generate_edge(bonds_source, bonds_target, edge, btypes):
                         [(0, 1), (0, 2), (2, 3), (3, 4), (2, 5), (2, 6), (4, 7),
                          (4, 8), (4, 9), (9, 10), (10, 11), (9, 12), (9, 13),
                          (11, 14), (11, 15), (11, 16), (16, 17)]),
+                        # smiple linear seqeunce with bond-order in link
+                        ("{[#TC1][#TC4][#TC1]}.{#TC1=[$1]=CC=[$2],#TC4=[$1]=CC=[$2]}",
```

Comment:
Should TC1 and TC4 be the same here?

Reply:
yeah that's fine; I initially had something else in mind but settled for this for the time begin.